zrml/prediction-markets/src/lib.rs

```diff
@@ -1232,6 +1299,24 @@ mod pallet {
         ValueQuery,
     >;
 
+    #[pallet::storage]
+    pub type MarketIdsPerOpenBlock<T: Config> = StorageMap<
+        _,
+        Blake2_128Concat,
+        T::BlockNumber,
+        BoundedVec<MarketIdOf<T>, ConstU32<1024>>,
```

Comment:
I think we can lower boundary here a bit because:
1. 1024 markets that close within a 6 second period is too high in my opinion
2. To reduce the effort required to state transit all the markets

If there should be the case that the boundary is surpassed, the user just defines the market to close 6 seconds earlier or later, effectively handling load balancing for the chain.

```suggestion
        BoundedVec<MarketIdOf<T>, ConstU32<32>>,
```

Reply:
If accepted, we should also change it for the closing caches.

Reply:
Yeah, I agree.

Reply:
(Mitigated in fc0555ed1d6d1c0f8c0672a47e8a96372a7db64e)

Reply:
You are right, there is not a single case on Battery Station where this occurs, so we can leave out the storage migration (since nobody else uses this pallet).

Reply:
Caches for closing fixed in 9644a79948f4152a9544b27497a0103c483fe032

Reply:
Also reduced the size of dispute/report caches in 079720bdafca679b84d898e796a234bb96186242